packages/utils/build-tools/src/transform-selector.ts

```diff
@@ -0,0 +1,10 @@
+import upstreamTransformer = require('metro-react-native-babel-transformer');
+import svgTransformer = require('react-native-svg-transformer');
+
+module.exports.transform = function({ src, filename, options }) {
```

Comment:
You should be able to convert this to:
```ts
export function transform({ src, filename, options }) {
```
rather than using the common JS style module syntax.

Reply:
using `export function transform({ src, filename, options }) {`
results in this error when bundling:

`@fluentui/react-native: [3:35:44 PM] x Error detected while running 'metro'
@fluentui/react-native: [3:35:44 PM] x ------------------------------------
@fluentui/react-native: [3:35:44 PM] x TypeError: transformer.transform is not a function
@fluentui/react-native:     at D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:231:31
@fluentui/react-native:     at Generator.next (<anonymous>)
@fluentui/react-native:     at asyncGeneratorStep (D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:75:24)
@fluentui/react-native:     at _next (D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:95:9)
@fluentui/react-native:     at D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:100:7
@fluentui/react-native:     at new Promise (<anonymous>)
@fluentui/react-native:     at D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:92:12
@fluentui/react-native:     at JsTransformer.transform (D:\repos\fluentui-react-native-fork\node_modules\metro\src\JSTransformer\worker.js:380:7)
@fluentui/react-native:     at Object.<anonymous> (D:\repos\fluentui-react-native-fork\node_modules\metro\src\DeltaBundler\Worker.js:102:38)`

Reply:
This seems to have just been some kind of weird build issue. It works now my last yarn build/bundle.